### PR TITLE
Feature/warmup speed

### PR DESCRIPTION
**Pull request by graetzer**
Loading each index in it's own thread + 2 read threads per edge index = 4 read Threads total

### CODE REVIEW

File: arangod/V8Server/v8-collection.cpp [hunk 1]
```diff
@@ -3298,12 +3300,19 @@ static void JS_WarmupVocbaseCol(
...
 
   for (auto& idx : idxs) {
-    // multi threading?
-    idx->warmup(&trx);
+    schdler->post([&] {
+      TRI_DEFER(numTrx--);
+      idx->warmup(&trx);
+    });
+  }
+  while (numTrx > 0) {
```
Comment (fceller) [inline arangod/V8Server/v8-collection.cpp]: need isStopping

File: arangod/RocksDBEngine/RocksDBEdgeIndex.cpp [hunk 2]
```diff
@@ -617,26 +619,85 @@ void RocksDBEdgeIndex::warmup(arangodb::transaction::Methods* trx) {
...
+  options.prefix_same_as_start = false;
+  options.total_order_seek = true;
+  options.verify_checksums = false;
+  options.fill_cache = EdgeIndexFillBlockCache;
+  std::unique_ptr<rocksdb::Iterator> it(rocksutils::globalRocksDB()->NewIterator(options, _cf));
+  
+  // get the first and last actual key
+  it->Seek(bounds.start());
+  if (!it->Valid()) {
+    LOG_TOPIC(INFO, Logger::ROCKSDB) << "Edge index is empty";
```
Comment (fceller) [inline arangod/RocksDBEngine/RocksDBEdgeIndex.cpp]: lower to debug

File: arangod/RocksDBEngine/RocksDBEdgeIndex.cpp [hunk 3]
```diff
@@ -617,26 +619,85 @@ void RocksDBEdgeIndex::warmup(arangodb::transaction::Methods* trx) {
...
+    return;
+  }
+  std::string firstKey = it->key().ToString();
+  it->SeekForPrev(bounds.end());
+  if (!it->Valid()) {
+    LOG_TOPIC(INFO, Logger::ROCKSDB) << "Edge index is empty";
```
Comment (fceller) [inline arangod/RocksDBEngine/RocksDBEdgeIndex.cpp]: lower to debug

File: arangod/RocksDBEngine/RocksDBEdgeIndex.cpp (hunk 2 above)
Comment (fceller) [inline arangod/RocksDBEngine/RocksDBEdgeIndex.cpp]: lower to debug (and following logs as well)

**Comment by graetzer** [comment]
@fceller I added the changes you requested

**Comment by fceller** [comment]
LGTM

**Pull request merged**